bindings/pubsubadapter/pubsubadapter.go

```diff
@@ -0,0 +1,150 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package pubsubadapter
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/components-contrib/pubsub"
+)
+
+const (
+	defaultReadTopicAttribute   = "topics"
```

Comment:
This is weirdly specific to the Kafka binding currently (i.e. there's no equivalent in MQTT, RabbitMQ etc. for taking a list of topics to listen on) and it doesn't look like it's actually used in the adapter, can it be dropped to simplify? It looks like the intention is for this to be breaking from the existing Kafka binding implementation anyway.